CHANGELOG.md

```diff
@@ -101,7 +103,7 @@
 * Restore fallback to full barrier when F_BARRIERSYNC is not available on Apple platforms. ([PR #6033](https://github.com/realm/realm-core/pull/6033), since v12.12.0)
 * Validation of Queries constructed by the Fluent QueryBuilder was missing. ([#6034](https://github.com/realm/realm-core/issues/6034), since v12.7.0)
 * Allow setting values on a Mixed property through the C API ([#5985](https://github.com/realm/realm-core/issues/5985), since v10.5.0)
- 
+
```

Comment:
Why is it that a ton of PRs that you do change a whole bunch of unrelated changelog entries? Is this something your editor is doing automatically?

Reply:
Sorry - it's trimming the whitespace at the end of lines. There was a whitespace in the template in the empty line after *Fixed*, which keeps getting trimmed. I can turn it off and I also updated the template to remove the whitespace to help eliminate this issue... (It also creates fun merges when I pull in master).